src/cc65/scanner.c

```diff
@@ -39,6 +39,7 @@
 #include <errno.h>
 #include <ctype.h>
 #include <math.h>
+#include <inttypes.h>
```

Comment:
I don't recall where/when this discussion was - but i would strongly prefer the stdint types being used in cases where bit size matters (and i will update contribute.md to reflect this right now)

Reply:
> and i will update contribute.md to reflect this right now

done. BTW, we have used RIX64 (and similar) macros before, so that wasn't a strict rule for a while

Reply:
As I said above, using fixed size data types is not necessary (which is true for most cases where people think otherwise).

But if you allow C99 features like inttypes.h, then the following should also be considered:

- `bool` data type
- // comments
- mixed statements and declarations

After all, I don't see any advantage in allowing some C99 but not others.

Reply:
> inttypes.h is not a new introduction to this codebase, there are many places using it already

Indeed

Reply:
Yes, seems you are correct. I interpreted the comments in #2049 as saying C99 features are unwanted. And it's easy to avoid in the given case.